pftools/python/parflow/tools/database/handlers.py

```diff
@@ -97,6 +101,17 @@ def decorate(self, value, container, class_name=None, location='.',
         raise ValueHandlerException(
             f'{value} is not of the expected type for ChildrenHandler')
 
+# -----------------------------------------------------------------------------
+class EnumLocationsHandler:
```

Comment:
I guess there are some confusion on what is a `handler` and a `domain`. Basically you can not be both hence the name of that class does not make any sense.

Reply:
The proper name should be `ListHandler`.

The responsibility of that handler is to ensure the value of that property get converted into a list. What you do with that list should be irrelevant to the handler...